lib/resources/dstu2/condition.yaml

```diff
@@ -0,0 +1,133 @@
+  example: |
+             {"encounter":{
+                "reference":"Encounter/1309819"}}
+
+- name: asserter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Person who takes responsibility for asserting the existence of the condition as part of the electronic record.
+  note: Asserter must be a `Practitioner` reference.
+  example: |
+             {"asserter":{
+                "reference":"Practitioner/2770007",
+                "display":"Song, River"}}
+
+- name: dateRecorded
+  required: 'No'
+  cardinality: 0..1
+  type: date
+  description: Estimated or actual date the condition/problem/diagnosis was first entered.
+  example: '{"dateRecorded":"2013-02-08"}'
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Identification of the condition, problem or diagnosis.
+  example: |
+             {"code":{
+                "coding":[{
+                  "system":"http://hl7.org/fhir/sid/icd-9",
+                  "code":"345.0",
+                  "display":"Generalized nonconvulsive epilepsy",
+                  "userSelected":true}],
+                "text":"Absence seizures"}}
+
+- name: category
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: A category assigned to the condition. e.g. complaint | symptom | finding | diagnosis.
+  example: |
+             {"category":{
+                "coding":[{
+                  "system":"http://hl7.org/fhir/condition-category",
+                  "code":"diagnosis",
+                  "display":"Diagnosis"}],
+                "text":"Diagnosis"}}
+
+- name: verificationStatus
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: The verification status of the condition.
+  example: '{"verificationStatus":"confirmed"}'
+
+- name: severity
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Assessment of the severity of the condition as evaluated by the clinician.
+  example: |
+             {"severity":{
+                "coding":[{
+                  "system":"http://snomed.info/sct",
+                  "code":"371923003",
+                  "display":"Mild to moderate"}],
+                "text":"Mild to moderate"}}
+
+- name: onsetDateTime
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Estimated or actual date the condition began, in the opinion of the clinician.
+  example: '{"onsetDateTime": "2015-10-14T13:13:20-06:00"}'
+
+- name: abatementBoolean
+  required: 'No'
+  cardinality: 0..1
+  type: boolean
+  description: Indicates if the condition is resolved or has gone into remission.
+  example: '{"abatementBoolean: true}'
+
+- name: abatementDateTime
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Indicates the date and time the condition was resolved or has gone into remission.
+  example: '{"abatementDateTime": "2015-10-14T13:13:20-06:00"}'
```

Comment:
Should we have some kind of note on these that either abatementBoolean OR abatementDateTime may be provided?